sbin/cubeit

```diff
@@ -376,6 +376,148 @@ install_summary()
 
 }
 
+#
+# Copy the installer 'static' parts, that is scripts, dirs,
+# files, libs...
+#
+# Configs are not static and so are dealt with as content
+# and copied later, see copy_installer_data().
+#
+copy_installer()
+{
+	local destdir="$1"
+
+	debugmsg ${DEBUG_INFO} "Copying installer to install media"
+	recursive_mkdir ${destdir}${INSTALLER_TARGET_LIB_DIR} || return 1
+
+	cp -r ${LIBDIR}/* ${destdir}${INSTALLER_TARGET_LIB_DIR}
+	if [ $? -ne 0 ]; then
+		debugmsg ${DEBUG_CRIT} "ERROR: Failed to copy lib files"
+		return 1
+	fi
+
+	recursive_mkdir ${destdir}${INSTALLER_TARGET_SBIN_DIR} || return 1
+
+	cp ${SBINDIR}/* ${destdir}${INSTALLER_TARGET_SBIN_DIR}
+	if [ $? -ne 0 ]; then
+		debugmsg ${DEBUG_CRIT} "ERROR: Failed to copy sbin files"
+		return 1
+	fi
+
+	# put the installers in with the sbin files
+	cp ${INSTALLERS_DIR}/* ${destdir}${INSTALLER_TARGET_SBIN_DIR}
+	if [ $? -ne 0 ]; then
+		debugmsg ${DEBUG_CRIT} "ERROR: Failed to copy installer files"
+		return 1
+	fi
+
+	# and any extended functions/callbacks
+	if [ -n "${FUNCTIONS_TO_COPY}" ]; then
+	    debugmsg ${DEBUG_INFO} "Copying target installer functions: ${FUNCTIONS_TO_COPY}"
+OLDIFS=$IFS
+IFS='
+'
+	    for f in ${FUNCTIONS_TO_COPY}; do
+		# remove any leading spaces
+		a=`echo $f | sed 's/^ *//g'`
+		if [ -e "${a}" ]; then
+		    cp -f "${a}" ${destdir}${INSTALLER_TARGET_SBIN_DIR}
+		fi
+	    done
+IFS=$OLDIFS
+	fi
+
+	## Copy files from local workspace to USB drive
+	recursive_mkdir ${destdir}${INSTALLER_TARGET_FILES_DIR} || return 1
+
+	## Copy the common files from ${INSTALLER_FILES_DIR} to ${INSTALLER_TARGET_FILES_DIR}
+	## Here only chvt.service, other files maybe added in the future
+	COMMON_FILES="chvt.service"
+	for f in ${COMMON_FILES}; do
+		if [ -e ${INSTALLER_FILES_DIR}/${f} ]; then
+			cp ${INSTALLER_FILES_DIR}/${f} ${destdir}${INSTALLER_TARGET_FILES_DIR}/
+		fi
+	done
+
+	recursive_mkdir ${destdir}${INSTALLER_TARGET_IMAGES_DIR} || return 1
+
+	return 0
+}
+
+#
+# Copies installer data (container images, dynamic configs...)
+#
+copy_installer_data()
```

Comment:
Is this 100% just putting the code into functions, or are there other changes within it ?